<launch>

    <!-- Node for naming the session ID -->
    <node name="talker" pkg="uav_pi" type="talker.py"/>

    <!-- Node for detecting objects -->
    <node name="object_detect_server" pkg="uav_pi" type="detect-pt.py"/>

    <!-- Node for processing detections -->
    <node name="detection_processor" pkg="uav_pi" type="detection_processor.py" output="screen"/>

    <!-- Gets gps data -->
    <node name="uav_info_publisher" pkg="uav_pi" type="uav_info_publisher.py" output="screen"/>

    <!-- Taking pics from the USB camera and sending to the topic-->
    <node name="pi_cam_publisher" pkg="uav_pi" type="pi_cam_publisher.py" output="screen"/>

    <!-- Service for publishing temperature data from the MLX -->
    <!-- <node name="MLX_publisher" pkg="uav_pi" type="MLX_publisher.py"/> -->
</launch>
